bumble/att.py

```diff
@@ -763,6 +783,24 @@ def read_value(self, connection):
         return self.encode_value(value)
 
     def write_value(self, connection, value_bytes):
```

Comment:
Since it's imported already:
`def write_value(self, connection: Connection, value_bytes: bytes):`